mhubert-pmn/Front-Url-Shortener
# [Feature] Create app routes

## Task
Create app routes

## Type of change

## Description
- Create routes
- Add components folder
- Add a folder for each view + the base of the js file
- Add layout folder
- Add the base of the js file for the header

## Testing scenario
**For now, it is possible to access to every view without being logged in.** There will be an update about this later.

In the terminal (you have to be in the front-url-shortener folder):
- Launch and build: `docker-compose up --build`
- Open the app at url `localhost:3000` you should see the title "Se connecter" (this is the sign in view)

Navigate to the other views:
- Sign up: `localhost:3000/inscription`
- Shorten an URL: `http://localhost:3000/raccourcir-une-url`
- See all the URL of a user: `http://localhost:3000/mes-urls`

- Down and delete running containers, volumes and networks: `docker-compose down`

## Trello card
[https://trello.com/c/0wxYEx3q](https://trello.com/c/0wxYEx3q)
